Dockerfile

```diff
@@ -277,20 +277,20 @@ RUN make distclean || echo "nothing to clean"
 RUN ./configure $GRASS_CONFIG \
     && make -j $NUMTHREADS \
     && make install && ldconfig \
-    &&  cp /usr/local/grass84/gui/wxpython/xml/module_items.xml module_items.xml; \
-    rm -rf /usr/local/grass84/demolocation; \
-    rm -rf /usr/local/grass84/fonts; \
-    rm -rf /usr/local/grass84/gui; \
-    rm -rf /usr/local/grass84/share; \
-    mkdir -p /usr/local/grass84/gui/wxpython/xml/; \
-    mv module_items.xml /usr/local/grass84/gui/wxpython/xml/module_items.xml;
+    &&  cp /usr/local/grass85/gui/wxpython/xml/module_items.xml module_items.xml; \
+    rm -rf /usr/local/grass85/demolocation; \
+    rm -rf /usr/local/grass85/fonts; \
+    rm -rf /usr/local/grass85/gui; \
+    rm -rf /usr/local/grass85/share; \
+    mkdir -p /usr/local/grass85/gui/wxpython/xml/; \
+    mv module_items.xml /usr/local/grass85/gui/wxpython/xml/module_items.xml;
```

Comment:
We could all replace them with a simple build ARG. What should it be called?

But in a docker image, could we assume a versionless path, as only one version is shipped in it? What is coupled to that path?
Below there is already a link that creates a versionless path.

Reply:
Please, follow on that in #3791. I don't plan to address that in this PR.